packages/jbrowse-web/src/rpc.worker.js

```diff
@@ -39,12 +40,12 @@ function wrapForRpc(func) {
   return args => {
     callCounter += 1
     const myId = callCounter
-    // logBuffer.push(['rpc-call', myId, func.name, ...args])
+    // logBuffer.push(['rpc-call', myId, func.name, args])
     const retP = Promise.resolve()
-      .then(() => func(jbPluginManager, ...args))
+      .then(() => func(jbPluginManager, args))
```

Comment:
This is intended, it's a bug fix in 23e873e4018bd1ec277a6e078b15c566ad7bac86 and 8c62a024760ff9e14d42119ce4e7f580cc650150. The timeout in `rpcManager.call` was actually being ignored before.

Also, it doesn't need an opts because the opts are for the rpcManager, not the function itself.